components/brave_news/browser/feed_v2_builder.h

```diff
@@ -32,6 +33,10 @@
 
 namespace brave_news {
 
+namespace {
+struct FeedGenerationInfo;
+}
```

Comment:
I think exposing anonymous namespace in header isn't better than private inner class, as it makes other fils that include this header have the same declaration in anonymous namespace

Reply:
Done!